docker/hibench-docker.conf

```diff
@@ -31,7 +31,7 @@ OS_VERSION                       14.04
 PYTHON_VERSION                   2.7
 JDK_VERSION                      7
 SCALA_VERSION                    2.10.5
-MAVEN_VERSION                    3.1.0
+MAVEN_VERSION                    3.0.5
```

Comment:
Because of the bad network environment, I use a local maven repository and  matched maven version(3.05). So I am only sure the docker can be built correctly in maven 3.05,